core/mock/goRoutineProcessorMock.go

```diff
@@ -0,0 +1,24 @@
+package mock
+
+import (
+	"bytes"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+type GoRoutineHandlerMap = map[string]core.GoRoutineHandler
```

Comment:
why redfining the GoRoutineHandlerMap type here?

Reply:
just so that the code looks better